### ISSUE TITLE
Missing context when users "share message"

### ISSUE THREAD

**Issue by abliss** [ltworf/localslackirc#130]
When users "share message" and add a comment, the RTM includes an "attachment" field on the message. The web client displays the text of this message, and the user sometimes adds a comment about it. This currently displays in IRC as a context-free comment since the attachment is dropped.
Not sure how this should be rendered. Maybe just insert a new message, as if sent by the sharing user, with the attachment in a quote?

**Comment by ltworf** [comment]
Do you know how the original slack irc gateway handled this?

**Comment by abliss** [comment]
No, I never used the original gateway. My impression is that they retired
it precisely because they wanted to add concepts like this that didn't fit
naturally into the irc model.

**Comment by daniel-Jones** [comment]
When a user shares a message with an attachment, the attachment appears in a random channel (in my case a private one where the file author isn't present) as if they just uploaded it there.

**Comment by ltworf** [comment]
The attachments seem to be deprecated in favour of a blocks, which are defined recursively :(
